lua/luaconf.h

```diff
@@ -842,18 +842,18 @@ extern void longjmp(luai_jmpbuf);
 #include <linux/ktime.h>
 #if LINUX_VERSION_CODE < KERNEL_VERSION(5,6,0)
 #include <linux/time.h>
-typedef struct timespec time_t;
+typedef struct timespec lunatik_time;
```

Comment:
I would just rename this to `lunatik_time_t` if it's alright to you..

Reply:
@lneto done https://github.com/luainkernel/lunatik/pull/23/commits/634dae30c1cd3315cae4d17331b7d7d0abc88619